tests/integration/features/callapi/recording.feature

```diff
@@ -116,3 +116,17 @@ Feature: callapi/recording
     And the response error matches with "call"
     Then user "participant1" starts "audio" recording in room "room1" with 400 (v1)
     And the response error matches with "call"
+
+  Scenario: Store recording
+    When the following "spreed" app config is set
+      | signaling_dev | yes |
+    And user "participant1" creates room "room1" (v4)
+      | roomType | 2 |
+      | roomName | room1 |
+    And user "participant1" joins room "room1" with 200 (v4)
+    And user "participant1" joins call "room1" with 200 (v4)
+    And user "participant1" starts "video" recording in room "room1" with 200 (v1)
+    And user "participant1" stops recording in room "room1" with 200 (v1)
+    Then user "participant1" store recording file "/img/join_call.ogg" in room "room1" with 200 (v1)
+    # And user "participant1" shares "join_call.ogg" with room "room1"
+    # And user "participant1" sees the following shared file in room "room1" with 200
```

Comment:
@nickvergessen how you think to do this?

I think on these possibilities:
* Send a boolean flag on POST to share or not the recording with room. If not, the manager will need go to `/Talk/Recording/<roomToken>/<filename> to share the file with room. This wouldn't a good UX to identify the place of recording.
* Implement an endpoint to return the list of recording files that wasn't shared with current room. This could be a bit borred to implement on clients.

Do you have other suggestion/ideas?

How these files will be managed (maybe deleted) if necessary? Going directly to folder Talk/Recording?

Reply:
It was agreed that the file is **NOT** shared into the talk room. See steps in https://github.com/nextcloud/spreed/issues/8326

Instead a notification should be generated with 2 actions:
* Primary: Share to chat
* Secondary: Dismiss notification

Reply:
Do you have an example of code that do this or have a similar behavior?

Reply:
I did this but I can't check if the notification is ok.
I never implemented a code doing notification.
A bit lost about notifications.

https://github.com/nextcloud/spreed/pull/8492/commits/9ceb2f4ecfaae726fa2761b2758260cad3956ff3

Reply:
Let's move the notifications to a follow up pull request, so we can get the endpoint in and Daniel can start working on the upload from the HPB

Reply:
Removed from this PR.
Waiting merge this to master to create the new branch and do the follow up.